src/pages/Donate.tsx

```diff
@@ -0,0 +1,234 @@
+                        <Typography
+                            variant="h3"
+                            component="h1"
+                            align="center"
+                            color="primary"
+                        >
+                            Donate
+                        </Typography>
+                    </div>
+                </div>
+                <div className={styles.container}>
+                    <Typography
+                        variant="body1"
+                        component="p"
+                        align="center"
+                        color="secondary"
+                    >
+                        If you'd like to donate funds to our cause, we would apply your kind donations towards
+                    </Typography>
+                    <Typography
+                        variant="body1"
+                        component="p"
+                        align="center"
+                        color="secondary"
+                    >
+                        keeping the platform up and running in the following ways:
+                    </Typography>
+                    <br></br>
+                </div>
+                <div className={styles.gridContainer}>
+                    <Grid container className={styles.root}>
+                        <Grid item xs={12} md={3}>
+                            <div className={styles.circle}>1</div>
+                            <Box className={styles.box}>
+                                Site hosting and server maintenance
+                            </Box>
+                        </Grid>
+                        <Divider orientation="vertical" flexItem={true} variant="middle" />
+                        <Grid item xs={12} md={3}>
+                            <div className={styles.circle}>2</div>
+                            <Box className={styles.box}>
+                                Ongoing improvements to site functions for providing increased value to members
+                                </Box>
+                        </Grid>
+                        <Divider orientation="vertical" flexItem={true} variant="middle" />
+                        <Grid item xs={12} md={3}>
+                            <div className={styles.circle}>3</div>
+                            <Box className={styles.box}>
+                                Platform security to protect the privacy of our community member
+                            </Box>
+                        </Grid>
+                    </Grid>
+                </div>
+                <div className={styles.container}>
+                    <br></br>
+                    <Typography
+                        variant="body1"
+                        component="p"
+                        align="center"
+                        color="secondary"
+                    >
+                        Any additional donations beyond deployment towards platform maintenance and
+                    </Typography>
```

Comment:
create a small component before return() to reduce code duplication, like:

const TextBody1 =  (content: string) => (
                  <Typography  ....>
                       {content}
                    </Typography>
)